apps/storybook/src/data-display/FileUpload.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import {
+  Box,
+  FileUpload,
+  FileUploadContext,
+  FileUploadDropzone,
+  FileUploadLabel,
+  FileUploadTrigger,
+  Flex,
+} from "@optiaxiom/react";
+
+const meta: Meta<typeof FileUpload> = {
+  component: FileUpload,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof FileUpload>;
+
+export const Primary: Story = {
+  render: () => {
+    return (
+      <Flex
+        alignItems="center"
+        asChild
+        border="1"
+        justifyContent="center"
+        rounded="lg"
+        w="320"
+      >
+        <FileUpload maxFiles={5}>
+          <FileUploadLabel>
+            <>
+              <h3>File Upload </h3>
+              <h5>Maximum 5 Files</h5>
+            </>
+          </FileUploadLabel>
+          <Flex
+            alignItems="center"
+            h="320"
+            justifyContent="center"
+            rounded="lg"
+            w="full"
+          >
+            <Flex
+              alignItems="center"
+              asChild
+              bg="bg.neutral"
+              border="1"
+              h="full"
+              justifyContent="center"
+              rounded="sm"
+              w="full"
+            >
+              <FileUploadDropzone>Drag your files here</FileUploadDropzone>
+            </Flex>
```

Comment:
We could wrap the components with `flex` or `Box` in export where we would just use the exported component and apply props, if necessary, rather than wrapping them in story.